doc/compiled.json

```
@@ -18301,16 +18301,89 @@
                 "title": "branch/merge/parameters",
                 "properties": {
                   "strategy": {
-                    "description": "strategy used for merge blocking, use_main or use_branch",
+                    "description": "strategy used for merge conflicts, use_main or use_branch",
                     "type": "string",
                     "example": "use_main"
                   }
                 }
               }
             }
           }
+        }
+      }
+    },
+    "/projects/{project_id}/branches/{name}/sync": {
+      "patch": {
+        "summary": "Sync a branch",
+        "description": "Sync an existing branch.\n\n*Note: Only available for branches created with New branching.*\n",
```

```suggestion
        "description": "Sync an existing branch.\n\n*Note: Only available for branches created with new branching.*\n",
```

also wondering if we should mention that it is in a private beta?